client/state/initial-state.js

```diff
@@ -82,7 +83,13 @@ export function persistOnChange( reduxStore, serializeState = serialize ) {
 			.catch( ( setError ) => {
 				debug( 'failed to set redux-store state', setError );
 			} );
-	}, SERIALIZE_THROTTLE, { leading: false, trailing: true } ) );
+	}, SERIALIZE_THROTTLE, { leading: false, trailing: true } );
+
+	if ( global.window ) {
```

Comment:
`global.window` is used a lot. See https://github.com/Automattic/wp-calypso/search?utf8=%E2%9C%93&q=global.window.